src/legacy/ui/public/field_editor/components/field_format_editor/editors/url/url.js

```diff
@@ -113,6 +120,42 @@ export class UrlFormatEditor extends DefaultFormatEditor {
     });
   }
 
+  renderWidthHeightParameters = () => {
+    const { error } = this.state;
+    const width = this.sanitizeNumericValue(this.props.formatParams.width);
+    const height = this.sanitizeNumericValue(this.props.formatParams.height);
+    return (
+      <Fragment>
+        <EuiFormRow
+          label={<FormattedMessage id="common.ui.fieldEditor.url.widthLabel" defaultMessage="Width" />}
+          isInvalid={!!error}
+          error={error}
+        >
+          <EuiFieldNumber
+            data-test-subj="urlEditorWidth"
+            value={width}
+            onChange={(e) => {
+              this.onChange({ width: e.target.value });
+            }}
+          />
+        </EuiFormRow>
+        <EuiFormRow
+          label={<FormattedMessage id="common.ui.fieldEditor.url.heightLabel" defaultMessage="Height" />}
+          isInvalid={!!error}
+          error={error}
```

Comment:
I might miss something, but how do I get into the error state? Might be a left-over of a former refactoring

Reply:
Good catch. Widht/height changes won't throw errors for now. I was thinking we should add some limitations on max/min values and then display errors accordingly. Not sure if it is necessary though and happy to remove these for now.